Docs/developer_guide/build_instructions/windows.md

```diff
@@ -56,20 +56,30 @@ Specify source, build, and Qt location and compiler version and start the build
 
 Release mode:
 
-```
+```bat
+cd C:\D
 mkdir C:\D\S4R
-cd /d C:\D\S4R
-"C:\Program Files\CMake\bin\cmake.exe" -G "Visual Studio 17 2022" -A x64 -DQt5_DIR:PATH=C:\Qt\5.15.2\msvc2019_64\lib\cmake\Qt5 C:\D\S4
-"C:\Program Files\CMake\bin\cmake.exe" --build . --config Release
+
+cmake ^
+  -G "Visual Studio 17 2022" -A x64 ^
+  -DQt5_DIR:PATH=C:/Qt/5.15.2/msvc2019_64/lib/cmake/Qt5 ^
+  -S C:\D\S4 -B C:\D\S4R
```

Comment:
For reference, since the minimum required version of CMake is 3.16, the `-S` and `-B` options are available. See https://cmake.org/cmake/help/v3.16/manual/cmake.1.html#options